src/mixins/Projector.ts

```diff
@@ -122,6 +123,18 @@ const eventHandlers = [
 	'onsubmit'
 ];
 
+/**
+ * Internal function that maps existing DOM Elements to virtual DOM nodes.
+ * @param vnode The virtual DOM node
+ * @param domNode The Element, if any, to set on the virtual DOM node
+ */
+function setDomNodes(vnode: VNode, domNode: Element | null = null) {
+	vnode.domNode = domNode;
+	if (vnode.children && domNode) {
```

Comment:
I'm not sure I get this check... meaning how does vnode.children exist and domNode not exist? It might be worth explaining this logic a bit better as I imagine it not being clear in 6 months.